internal/releaseversion/spec.go

```diff
@@ -0,0 +1,11 @@
+package releaseversion
+
+import (
+	"github.com/blang/semver"
+	"github.com/giantswarm/apiextensions/v2/pkg/apis/release/v1alpha1"
+)
+
+type release struct {
```

Comment:
@tuommaki I added this internal struct so there are no two slices that are passed around (`[]semver.Version` and `[]v1alpha1.Release`), but one with both info bundled together so each can be used where needed.